closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -52,6 +52,13 @@ else ifeq ($(call isTargetOs, windows), true)
     OPENJCEPLUS_JGSKIT_MAKE := jgskit.win64.mak
     OPENJCEPLUS_JGSKIT_PLATFORM := win64
   endif
+else ifeq ($(call isTargetOs, macosx), true)
+  ifeq ($(call isTargetCpu, x86_64), true)
+    OPENJCEPLUS_JGSKIT_PLATFORM := x86_64-mac
+  else ($(call isTargetCpu, aarch64), true)
+    OPENJCEPLUS_JGSKIT_PLATFORM := aarch64-mac
+  endif
+  OPENJCEPLUS_JGSKIT_MAKE := jgskit.mac.mak
```

Comment:
Please keep the target cases sorted; and for macOS, keep the architectures sorted:
```
else ifeq ($(call isTargetOs, macosx), true)
  ifeq ($(call isTargetCpu, aarch64), true)
    OPENJCEPLUS_JGSKIT_PLATFORM := aarch64-mac
  else ifeq ($(call isTargetCpu, x86_64), true)
    OPENJCEPLUS_JGSKIT_PLATFORM := x86_64-mac
  endif
  OPENJCEPLUS_JGSKIT_MAKE := jgskit.mac.mak
else ifeq ($(call isTargetOs, windows), true)
```